src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,41 @@ metadata:
     custom.label/dev: "true"
 ```
 
+#### mode (string, optional)
+
+Defines how `okteto up` will setup the development container and behave within the cluster.
+
+Currently two options are available:
+- `sync`: (default) Sync & run your code in a remote development container.
+- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
+
+Most use-cases should use default `sync` mode, but `hybrid` may be desired to achieve a faster development experience in some scenarios:
```

Comment:
This needs to be more specific. Can we talk more concretely about why pick one vs the other? If not, I think it's better to remove this and describe the options without having an opinion on which one to use.

Reply:
should we talk about it on the dedicated page/blog?

Reply:
I agree with @rberrelleza that we should remove it here if we don't explain it in detail. More thoughts here: https://github.com/okteto/docs/pull/447/files#r1259847016